models/review.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+const { ObjectId } = mongoose.Schema
+
+const reviewSchema = new mongoose.Schema({
+  userId: { type: ObjectId },
+  businessId: { type: ObjectId },
```

Comment:
Hi @cynthiaeddy! Once the feature is finalized, do you think there is value in making the `userId` and `businessId` required?

Reply:
shouldn't affect this pr - once i figure out how to have the reviews show up in the business collection this will be resolved.